src/App.tsx

```diff
@@ -15,49 +17,116 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum ButtonType {
+  ALPHABET = 'is-info',
+  LENGTH = 'is-success',
+  REVERSE = 'is-warning',
+  RESET = 'is-danger',
```

Comment:
It would be better to name it like:
```suggestion
  INFO = 'is-info',
  SUCCESS = 'is-success',
  WARNING = 'is-warning',
  DANGER = 'is-danger',
```
because these types are related to this component, and for example you want to use another button in another component that would be 'Buy', and you will set type={ButtonType.LENGTH}, it can be strange